circlemenu/src/main/res/values/attrs.xml

```diff
@@ -18,6 +18,8 @@
         <attr name="isRotating" format="boolean" />
         <attr name="radius" format="dimension" />
         <attr name="speed" format="integer" />
+        <attr name="wheel_radius_ratio" format="integer" />
```

Comment:
why does it has to be an integer? could be float

Reply:
you are right,